alembic/env.py

```diff
@@ -0,0 +1,78 @@
+from logging.config import fileConfig
+from sqlalchemy import engine_from_config, pool
+from alembic import context
+from database import Base
+from city.models import City
+from temperature.models import Temperature
```

Comment:
Ensure that the `Temperature` model is correctly defined and imported from `temperature.models`. This is important for Alembic's autogenerate feature to detect changes.